Versioning.md

```diff
@@ -19,38 +19,38 @@ must remain backwards compatible. Internal types are allowed to break.
 
 ### ABI Stability
 
-Refer [ABI Policy](./docs/abi-policy.md) for more details. To summarise
-* ABI stability is guaranted for API.
-* ABI stability is not guaranteed for SDK. C++ In case of ABI breaking changes, instead of bumping up the majore version, new `inline namespace` version will be crearted, and both old API and new API would be made available simultaneously.
+Refer to the [ABI Policy](./docs/abi-policy.md) for more details. To summarise
+* ABI stability is guaranteed for the API.
+* ABI stability is not guaranteed for the SDK. In case of ABI breaking changes, instead of bumping up the major version, a new `inline namespace` version will be created, and both old API and new API would be made available simultaneously.
 
 ## Policy
 
 * Release versions will follow [SemVer 2.0](https://semver.org/).
-* Only single source package containing api and sdk for all signals would be released as part of the new GitHub release.
-* There would be source package releases for api and sdk. There won't be separate releases for the signals. The release version numbers for api and sdk will not be in sync with each other. As there would be more frequent changes expected in sdk then in api.
-* Experimental releases: New (unstable) telemetry signals and features will be introduced behind feature flag using C macro define.
+* Only a single source package containing both the api and sdk for all signals will be released as part of each GitHub release.
```

Comment:
API and SDK are styled inconsistently, consider using the same case (`api` vs. `API`) throughout the document. Consider also defining or referencing a definition of what the API vs. the SDK is.

Reply:
Will add that. Thanks for noticing it.